src/notation/internal/notationinteraction.cpp

```diff
@@ -1150,8 +1153,27 @@ void NotationInteraction::startDragCopy(const EngravingItem* element, QObject* d
         m_drag = nullptr;
     });
 
-    static QPixmap pixmap(2, 2); // null or 1x1 crashes on Linux under ChromeOS?!
+    const qreal adjustedRatio = 0.4;
+    const qreal width = element->ldata()->bbox().width();
+    const qreal height = element->ldata()->bbox().height();
+
+    QSize pixmapSize = QSize(width * adjustedRatio, height * adjustedRatio);
+    QPixmap pixmap(pixmapSize); // null or 1x1 crashes on Linux under ChromeOS?!
     pixmap.fill(Qt::white);
+    QBitmap mask = pixmap.createMaskFromColor(Qt::white); // Transparent background
+    pixmap.setMask(mask);
```

Comment:
I wonder if this can't be simplified to 
```suggestion
    QPixmap pixmap(pixmapSize);
    pixmap.fill(Qt::transparent);
```

(Also, I think the comment "// null or 1x1 crashes on Linux under ChromeOS?!" is not necessary anymore; it just explained why a value of 2x2 was used.)